bentoml/_internal/io_descriptors/image.py

```diff
@@ -140,7 +141,9 @@ async def predict_image(f: Image) -> NDArray[Any]:
     def __init__(
         self,
         pilmode: _Mode | None = DEFAULT_PIL_MODE,
-        mime_type: str = "image/jpeg",
+        mime_type: str | t.Iterable[str] | None = None,
```

Comment:
I think this should be a `t.Sequence` instead. str is also considered `t.Iterable`
```suggestion
        mime_type: str | t.Sequence[str] | None = None,
```

Reply:
I think `str` is also a sequence, but we can remove the `str` type hint.

Reply:
Actually, I think it's better for documentation to allow `str` explicitly.

Reply:
We don't want to make this only accept mutable sequences. I think the current types are fine.

Reply:
noted.